package.json

```diff
@@ -24,6 +26,11 @@
     "storybook:clean": "rimraf storybook-static",
     "release": "semantic-release"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "yarn lint:fix && yarn lint"
```

Comment:
What's the reason for running both lint fix and lint?

Reply:
I thought it would fix any lint errors, and finally check to see if there are still errors that it couldn't fix... I just tested it, and it doesn't work :/ woops.. using lint-staged now :)